app/Models/Group.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+
+class Group extends Model
+{
+
+    protected $table = 'group';
+
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = [
+        'name',
+        'location',
+        'tractor_ids',
```

Comment:
Does this mean that a group could have more than one tractor?

Reply:
Yes, assuming they get one and pay for it. Then need a loan to get a different type of tractor. Will they need to create a new group?